packages/ag-charts-community/src/chart/themes/util.ts

```diff
@@ -0,0 +1,16 @@
+import type { AgCartesianChartOptions } from 'ag-charts-types';
+
+type CartesianAxis = Exclude<AgCartesianChartOptions['axes'], undefined>[0];
+
+export function swapAxisCondition(axes: [CartesianAxis, CartesianAxis], swap: (series: any) => boolean) {
+    return (series: any) => {
+        if (swap(series)) {
+            return [
+                { ...axes[0], position: axes[1].position },
+                { ...axes[1], position: axes[0].position },
+            ];
+        } else {
+            return axes;
+        }
```

Comment:
Nit: `else` is redundant, just drop this block and `return axes;` after the `if` block close?